engines/wintermute/base/base_keyboard_state.cpp

```diff
@@ -246,18 +247,22 @@ bool BaseKeyboardState::readKey(Common::Event *event) {
 	}
 
 	// use keyCodeToVKey mapping for all other events
-	// some keys are printable from those keys
-	else {
+	// in WME 1.x some of those keys are printable
+	else if (BaseEngine::instance().getTargetExecutable() < WME_LITE) {
 		_currentCharCode = keyCodeToVKey(event);
 		_currentPrintable = code == Common::KEYCODE_BACKSPACE || 
-		                    code == Common::KEYCODE_TAB || 
-		                    code == Common::KEYCODE_RETURN || 
-		                    code == Common::KEYCODE_KP_ENTER || 
-		                    code == Common::KEYCODE_ESCAPE || 
-		                    code == Common::KEYCODE_SPACE;
+							code == Common::KEYCODE_TAB || 
+							code == Common::KEYCODE_RETURN || 
+							code == Common::KEYCODE_KP_ENTER || 
+							code == Common::KEYCODE_ESCAPE;
```

Comment:
It's usually a bad idea to align text using tabs, since tab spacing is a user configured setting in each editor. Nevertheless, since tabs are used for spacing throughout this file, this should be OK